doc/extended-configuration.md

```diff
@@ -140,6 +140,14 @@ You may want to provide a cron expression to configure how frequently the ip-rec
 
 Look for the following parameter `"reconciler_cron_expression"` located in `script/install-cni.sh` and change to your desired schedule.
 
+alternatively you can speficy the `WHEREABOUTS_RECONCILER_CRON` environment variable in your daemonset definition file to provide the cron expresion.
+
+```
+       env:
+        - name: WHEREABOUTS_RECONCILER_CRON
+          value: 30 * * * *
+```
```

Comment:
After re-reading this, I would actually even omit the other option (now that you're provided us a neat API to propagate the cron expression).

```suggestion
You can speficy the `WHEREABOUTS_RECONCILER_CRON` environment variable in your daemonset definition file to override the default cron expression:
```yaml
       env:
        - name: WHEREABOUTS_RECONCILER_CRON
          value: 30 * * * *
```
```